README.md

```diff
@@ -72,3 +72,9 @@ You might have some conflicts while rebasing, try to resolve the conflicts for e
 In order to merge a PR, it will first go through a review process. Once it is approved, we will merge to the `development` branch using the `Squash` button in github.
 
 When using squash, all the commits will be squashed into one. The idea is to merge features/fixes as oppose of merging each individual commit. This helps when looking back in time for changes in the code base, and if the PR has a great comment, it's easier to know why that code was introduced.
+
+### Google Analytics
+
+Create a `.env` file in the root directory. Add your Tracking ID under the variable `REACT_APP_GA_TRACKING_ID` e.g:
+
+    REACT_APP_GA_TRACKING_ID='UA-xxxxxxxxx-x'
```

Comment:
I think we should add a sample `.env` like `.env.sample` with all the ids and commit to git but with empty or demo values. This is done so that all the devs are in sync about all the env used in the app. Or maybe we can create a separate markdown file mentioning about all the env variables (just the varibale names, not the key values) used in the app.

Reply:
Yea I think it's good for contributors to have some default keys to quickly get up and running

Reply:
I think having an `.env.example` is a good idea so all the devs know what env vars there are

Reply:
Ok I'll add an `.env.example` file with a GA Tracking so everyone can get up and running smoothly.